src/solvers/dg_unstructured_quad/dg_element_geometry_2d.jl

```diff
@@ -0,0 +1,11 @@
+# construct the inverse Jacobian for a given element (this is agnostic as to the curvature)
+# TODO: this could be adapted to reuse the routine already persent in CurveMesh
+function calc_inverse_jacobian!(inverse_jacobian::AbstractArray{<:Any, 3}, element_id, X_xi, X_eta,
+                                Y_xi, Y_eta)
+  @. @views inverse_jacobian[:, :, element_id] = inv(X_xi[:, :, element_id] * Y_eta[:, :, element_id] -
+                                                     X_eta[:, :, element_id] * Y_xi[:, :, element_id])
```

Comment:
Does this do the right thing? Do you want to calculate the inverse of the matrix
```julia
X_xi[:, :, element_id] * Y_eta[:, :, element_id] - X_eta[:, :, element_id] * Y_xi[:, :, element_id]
```
or the matrix with entries
```julia
inv(X_xi[i, j, element_id] * Y_eta[i, j, element_id] - X_eta[i, j, element_id] * Y_xi[i, j, element_id])
```
Some comment might be good

Reply:
I just want to compute 1/jacobian entries. Maybe I confused myself with the naming...